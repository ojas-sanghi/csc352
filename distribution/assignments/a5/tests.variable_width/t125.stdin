
 ofae   rmtnymp  ns  uj  ofn  omuq    atheiihd  ocioci usmpa   
    msw    ervruuv   nugzna  bbspynt    tixmm    xupcnk vq 



  zyodscj    owsl  dq  clsmfxl  vqdsexf  pyjiolgq lsie   wniita 


 ay kswg bupqc   dsn  gt  uacishe   igzx  ke rz  vqnjcak   byshs  qta  inklp    gxqykija    xbiotwv   kftphnxc    oon   eqocwgtp   hbw pl qngyvc    
