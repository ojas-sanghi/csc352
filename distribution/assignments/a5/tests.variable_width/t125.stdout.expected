    ofae  bbspynt      ay    inklp
 rmtnymp    tixmm    kswg gxqykija
      ns   xupcnk   bupqc  xbiotwv
      uj       vq     dsn kftphnxc
     ofn  zyodscj      gt      oon
    omuq     owsl uacishe eqocwgtp
atheiihd       dq    igzx      hbw
  ocioci  clsmfxl      ke       pl
   usmpa  vqdsexf      rz   qngyvc
     msw pyjiolgq vqnjcak
 ervruuv     lsie   byshs
  nugzna   wniita     qta
